sandpack-react/src/contexts/sandpackContext.tsx

```diff
@@ -394,7 +423,8 @@ class SandpackProvider extends React.PureComponent<
           this.unsubscribeQueuedListeners[clientId] || {};
 
         this.queuedListeners[clientId][listenerId] = listener;
-        return () => {
+
+        const unsubscribeListener = () => {
```

Comment:
Question: Is this useful for debugging or why is the change needed?

Reply:
I did it just to make the code clearer, now each conditional returns the same kind of function and which makes it easy to read long complex parts of code.